aio/content/examples/testing/src/app/hero/hero-detail.component.spec.ts

```diff
@@ -107,6 +107,7 @@ function overrideSetup() {
     const newName = 'New Name';
 
     page.nameInput.value = newName;
+    // newEvent is not a typo, there is a newEvent helper function that's in the live example
```

Comment:
How about something like this?

```suggestion
    // newEvent() is a helper function defined in testing/src/testing/index.ts that creates custom DOM events
```

Reply:
okay . . . I'm going to take another attempt at it.